qa/tasks/persistent_writeback_cache_thrash.py

```diff
@@ -0,0 +1,76 @@
+"""
+Persistent writeback cache recovery test
+"""
+import logging
+import contextlib
+from teuthology import misc as teuthology
+from teuthology import contextutil
+from teuthology.orchestra import run
+
+log = logging.getLogger(__name__)
+
+def run_rbd_bench(ctx, client, image, is_bg):
+    (remote,) = ctx.cluster.only(client).remotes.keys()
+    cmd = [
+           "rbd",
+           "bench",
+           "--io-type",
+           "write"
+           ]
```

Comment:
Nit: maybe `--io-pattern rand` to avoid simple sequential case

Reply:
DONE